solution/ui/regulations/css/scss/partials/_related_sections.scss

```diff
@@ -0,0 +1,35 @@
+//----------------------------------
+// Related Sections component styles
+//----------------------------------
+
+@import "../application_settings";
+
```

Comment:
Related Sections component is now used across three pages (resources, search, policy repository) so it deserves its own stylesheet.